Trendyol.App/Trendyol.App.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Trendyol.App</id>
-    <version>2.4.3</version>
+    <version>2.4.4</version>
```

Comment:
You don't need to increase this version number since there is no changes in this project.